index.js

```diff
@@ -53,22 +53,11 @@ renderSales()
 // 3. update "Live Sales" with times either of the two buttons is clicked
 // 4. update totalRevenue and totalCommision with values set in respective product arrays
 // 5. update totalRevenue and totalCommision in the HTML with the content of their arrays
-function salesA() {
+function fixSale(object) {
```

Comment:
This works great, combining a lot of the repeated code into a unified function. You might consider the parameter name here (object), possibly changing to something that makes it clear what sort of data the function expects (product, productObj). That's maybe more of a personal convention on my part though. Works well either way!